lms/templates/help_modal.html

```diff
@@ -123,25 +122,18 @@ <h2>${_('Thank You!')}</h2>
           <hr>
         </header>
 
-    <%
-      dst = datetime.now(pytz.utc).astimezone(pytz.timezone("America/New_York")).dst()
-      if dst:
-        open_time = "13:00"
-        close_time = "21:00"
-      else:
-        open_time = "14:00"
-        close_time = "22:00"
-    %>
-    <p>
-    ${Text(_(
-      'Thank you for your inquiry or feedback. We typically respond to a request within three '
-      'business days. In the meantime, please review our {link_start}Help Center articles{link_end}, '
-      'where answers to many common questions have been posted.'
-    )).format(
-        link_start=HTML(u'<a href="{}" target="_blank" id="feedback-faq-link" tabindex="0">').format('https://stanfordonline.zendesk.com/hc/en-us'),
-        link_end=HTML(u'</a>')
-    )}
-    </p>
+        <p>
+        ${Text(_(
+            'Thank you for your inquiry or feedback.  We typically respond to a request '
+            'within three business days, Monday to Friday.  In the meantime, please '
```

Comment:
In the future, we could move the number of days into a variable and send this upstream.